go/tasks/pluginmachinery/internal/remote/allocation_token.go

```diff
@@ -0,0 +1,88 @@
+package remote
```

Comment:
Can we think about this name "remote"?  I like the idea of service or something.  A call to create a K8s pod is also remote right?  Like, it's not done in Propeller or in the plugins - it has to contact the K8s API server, which is remote.  I feel like the defining feature of this whole concept is not that it's remote but rather that it's a non-Flyte controlled service that creates some entities on behalf of Flyte.  Should/could this framework be used to interact with external services that don't offer a read/get endpoint?  What about an external service that doesn't offer read after write consistency?

Reply:
I know that was a long time ago... I'm finally coming back to this... Any suggestions? service also applies to k8s resources (we call ApiServer which is, at the end of the day, a service)... I don't feel strongly either way...

Bringing @kumare3 to the naming fun discussion...